safe_cli/operators/hw_accounts/hw_account_manager.py

```diff
@@ -0,0 +1,140 @@
+from enum import Enum
+from typing import Dict, List, Optional, Set, Tuple
+
+from eth_typing import ChecksumAddress
+from prompt_toolkit import HTML, print_formatted_text
+
+from gnosis.eth.eip712 import eip712_encode
+from gnosis.safe import SafeTx
+
+from safe_cli.operators.hw_accounts.hw_account import HwAccount
+
+
+class HwWalletType(Enum):
+    TREZOR = "Trezor"
+    LEDGER = "Ledger"
```

Comment:
I find this a little weird. Usually Enums map to numeric values, not to strings. I would rather have it that way unless you have a reason for that